### ISSUE TITLE
Compound Tooltips lack labels

### ISSUE THREAD

**Issue by t3chguy** [element-hq/compound#322]
They add an `aria-describedby` on focus but lack any label at any other time which is not great for a11y technologies, see https://sarahmhigley.com/writing/tooltips-in-wcag-21/.

The only way around this from the caller is to specify `aria-label` on the child of the Tooltip which then means the label & description may be read, both containing the same text content.

**Comment by robintown** [comment]
I'll see if I can knock this out today (timeboxed)
